tequilapi/client/dto.go

```diff
@@ -82,3 +82,23 @@ type BuildInfoDTO struct {
 	Branch      string `json:"branch"`
 	BuildNumber string `json:"buildNumber"`
 }
+
+// RegistrationStatusDTO holds input data required to register new myst identity on blockchain smart contract
+type RegistrationStatusDTO struct {
```

Comment:
Your tequitelapi client-side DTOs is not compatible with Tequilapi service-side DTOs, see `RegistrationDataDTO`
Suggest to add JSON annotations

Reply:
E2e test rely heavily on tequilapi client - in case of incomaptible DTO (response parsing), it would fail with fireworks. Anyway - added json annotations for the sake of clarity.